src/diffusers/utils/outputs.py

```diff
@@ -60,11 +61,18 @@ def __init_subclass__(cls) -> None:
         if is_torch_available():
             import torch.utils._pytree
 
-            torch.utils._pytree._register_pytree_node(
-                cls,
-                torch.utils._pytree._dict_flatten,
-                lambda values, context: cls(**torch.utils._pytree._dict_unflatten(values, context)),
-            )
+            if version.parse(get_torch_version()) < version.parse("2.2"):
```

Comment:
Can we not use something like

https://github.com/huggingface/diffusers/blob/8974c50bff1f51c5a1208723af88d4b4ed7f2b16/src/diffusers/pipelines/pipeline_utils.py#L1221

?

Reply:
Hi @sayakpaul ! I have addressed this, please lmk if it's alright, cheers!